iaso/utils/models/common.py

```diff
@@ -11,3 +11,14 @@ def get_creator_name(creator: User = None, username: str = "", first_name: str =
     elif username:
         return username
     return ""
+
+
+def get_org_unit_parents_ref(field_name, org_unit, parent_source_ref_field_names, parent_field_ids):
+    if org_unit.get(field_name):
+        return org_unit.get(field_name)
+    else:
+        parent_index = parent_source_ref_field_names.index(field_name)
+        parent_ref = org_unit.get(parent_field_ids[parent_index])
+        if parent_ref:
+            return f"iaso#{parent_ref}"
```

Comment:
Why are you prefixing `parent_ref` with `iaso#` ?
This could be explained in a comment so that the next person to read the code will understand.